### PR TITLE
rtt localization for Tenet Chapel B1

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -5082,39 +5082,39 @@ QUEST_LV_0100_20150317_005077	Follower Donatas told you to check the central bar
...
-QUEST_LV_0100_20150317_005088	Defeat Rodelin and obtain %s
+QUEST_LV_0100_20150317_005086	Follower Tomas told you to defeat Rodelins and retrieve their Eyes of Madness, which can be used to hide from demons.
+QUEST_LV_0100_20150317_005087	You collected enough Eyes of Madness. Return to Follower Tomas.
+QUEST_LV_0100_20150317_005088	Defeat Rodelins and obtain %s
 QUEST_LV_0100_20150317_005089	Obtain the Eyes of Madness
-QUEST_LV_0100_20150317_005090	Follower Tomas is seeking for somebody's help on the 1st floor of Tenet Chapel.
-QUEST_LV_0100_20150317_005091	Follower Tomas told you to defeat Unknocker beside the barrier first. Attack Unknocker when Follower Tomas lures it.
-QUEST_LV_0100_20150317_005092	You defeated the Unknocker that threatened the barrier. Talk to Follower Tomas again.
+QUEST_LV_0100_20150317_005090	Follower Tomas wants help on the 1st floor of Tenet Chapel.
+QUEST_LV_0100_20150317_005091	Follower Tomas told you to defeat the Unknocker that is guarding the altar needed to disable the barrier. Attack Unknocker when Follower Tomas lures it.
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Follower Tomas told you that it is necessary to defeat the Unknocker that is guarding the altar in order to disable the barrier.

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -780,24 +780,24 @@ QUEST_LV_0100_20150317_000775	At the very least, the demons should stop crossing
...
+QUEST_LV_0100_20150317_000785	The Yognomes are trying to eat the altar.{nl}I want you to defeat them.
+QUEST_LV_0100_20150317_000786	If I had any Holy Stones, I would stick them in the Yognomes' mouths.
 QUEST_LV_0100_20150317_000787	Follower Tiberius
-QUEST_LV_0100_20150317_000788	I am making a lethal bomb to attack demons.{nl}It can be done if I have a Purified Essence. Can you help me?
+QUEST_LV_0100_20150317_000788	I am making a Holy Bomb to attack demons with.{nl}I can complete it if I obtain Purified Essence. Can you help me?
 QUEST_LV_0100_20150317_000789	Bombs in the chapel doesn't sound good, but..{nl}I think the Goddess will forgive me this time.
-QUEST_LV_0100_20150317_000790	I want to test it on Glizardon.{nl}I'll give you the bomb, so that you can put it onto the back of Glizardon?
-QUEST_LV_0100_20150317_000791	If you think it'll be dangerous, I will also give you Romuva Divine Water.{nl}When you drink this water, those demons will be invisible for a while.
+QUEST_LV_0100_20150317_000790	I want to test it out on Glizardon.{nl}I'll give you the Holy Bomb.  Can you put it onto a Glizardon's back?
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: There is an extra space before "Can you put it onto..."

**Comment by terrapin47** [comment]
:+1:

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -877,8 +877,8 @@ QUEST_LV_0100_20150317_000872	If you don't mind, can you get Eyes of Madness fro
 QUEST_LV_0100_20150317_000873	I don't want those filthy things to be near me either.{nl}But, it's worse to face against those demons alone.
 QUEST_LV_0100_20150317_000874	Thanks.{nl}I will share them with the other brothers before the demons find out.
 QUEST_LV_0100_20150317_000875	You are indeed the revelator.{nl}Now, we can concentrate on protecting the altar.
-QUEST_LV_0100_20150317_000876	Don't worry, the barrier of the main entrance on the 1st floor can be offset by the power of the chapel. {nl}That's what the altars are for. {nl}
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:880]: power of the chapel or power of the altar? o-O
so the Korean text apparently is correct for this and the other line is also correct. xD

I would say: Don't worry, the barrier at the main entrance to the 1st floor can be offset by the chapel's power.

Comment (terrapin47) [inline QUEST_LV_0100.tsv:880]: It seems to say that the altars are used to channel the power of the chapel, or some such.

Comment (ttgmichael) [inline QUEST_LV_0100.tsv:880]: Yep, I understand that part; It's just the way the term is phrased could throw some people off thinking altar and chapel are the same thing.

Comment (terrapin47) [inline QUEST_LV_0100.tsv:880]: Fixed

**Comment by ttgmichael** [comment]
good to merge :+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**